src/com/twitter/intellij/pants/file/FileChangeTracker.java

```diff
@@ -129,7 +131,15 @@ private static ChangeType detectChangeType(Project project, @NotNull VirtualFile
     if (rootManager.getFileIndex().getContentRootForFile(file) != null) {
       boolean isBuildFile = PantsUtil.isBUILDFileName(file.getName());
       boolean isUnderPantsRepo = PantsUtil.isFileUnderPantsRepo(file);
-      if (isUnderPantsRepo && isBuildFile) {
+
+      Path filePath = Paths.get(file.getPath());
+      boolean shouldBeIgnored =
+        PantsOptions.getPantsOptions(project)
+          .flatMap(options -> options.get("pants_workdir"))
```

Comment:
can we move `pants_workdir` to a constant? https://github.com/pantsbuild/intellij-pants-plugin/blob/master/common/com/twitter/intellij/pants/util/PantsConstants.java#L54

Reply:
Fixed!